### ISSUE TITLE
Communicate SIGINT to the parent shell

### ISSUE THREAD

**Issue by UnkindPartition** [UnkindPartition/tasty#309]
See <https://www.cons.org/cracauer/sigint.html>

**Comment by santiweight** [comment]
Heya - I'm havingt his issue quite often and it's really annoying - is there somewhere you could point me to in the code to fix it?

**Comment by UnkindPartition** [comment]
Yeah, I had a branch with a fix, but something funky was going on when I ran an automated test for it...

I'll try to dig up and push that branch next time I have access to the computer (https://github.com/UnkindPartition/UnkindPartition/blob/main/README.md) and maybe someone can finish it.
